tests/functional/cursorid-001.phpt

```diff
@@ -16,16 +16,14 @@ $query = new MongoDB\Driver\Query(array(), array(
     'limit' => 110,
 ));
 
-$qr = $manager->executeQuery(NS, $query);
-$cursor = $qr->getIterator();
+$cursor = $manager->executeQuery(NS, $query);
+
 $cursorid = $cursor->getId();
 $s1 = (string)$cursorid;
 var_dump(
     $cursorid,
     $s1
 );
-$cursor->kill();
```

Comment:
This method was removed. Killing of cursors when they go out of scope will be implemented in [PHPC-225](https://jira.mongodb.org/browse/PHPC-225).